Tests/Admin/AdminTest.php

```diff
@@ -846,6 +851,19 @@ public function testGetBaseCodeRoute()
         $this->assertSame('foo', $admin->getBaseCodeRoute());
     }
 
+    // NEXT_MAJOR: uncomment this method.
+    // public function testGetBaseCodeRoute()
```

Comment:
I'm a bit undecided about this. It does indeed create merge conflicts, but postponing the work often means it will have to be done by a "cleaner" (a.k.a a Sonata member) in one big, and thus more error-prone cleaning pass, rather by the contributor that actually wrote the PR, and is in a better position to know what to do. I think we may instead ask our contributors to produce code that is very easy to clean. Ideally:

- the deprecated path should be isolated from the recommended path;
- the cleaning should consist in removing the deprecated path; and should not require editing the recommended path.